### PR TITLE
BigAnimal: Updates to EHA content for PGE 14 & 15

### PR DESCRIPTION
**Pull request by ccestes**
## What Changed?

Adding PostgreSQL Compatibility for PGE 14 & 15 with EHA clusters

### CODE REVIEW

**Review by drothery-edb — COMMENTED** [review]
LGTM

File: product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ We support the major Postgres versions from the date they're made available unti
 | ---------------------------- | --------------------------------------------------- |
 | PostgreSQL                   | 11–15                                               |
 | EDB Postgres Advanced Server | 11–15, 14-15 for extreme-high-availability clusters |
-| EDB Postgres Extended Server | 12-15                                               |
+| EDB Postgres Extended Server | 12-15, 14-15 for extreme-high-availability clusters |
```
Comment (aaronsonntag) [inline product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx:13]: I don't believe we actually support PGE for Single and HA offerings?  We will soon though?
```suggestion
| EDB Postgres Extended Server | 14-15 for extreme-high-availability clusters |
```

**Review by aaronsonntag — COMMENTED** [review]
Suggesting change to update version policy.  Unless I've misunderstood?

File: product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx (hunk 1 above)
Comment (ccestes) [reply product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx:13]: Okay going to commit this

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**